android/src/main/java/com/robinpowered/RNMDMManager/RNMobileDeviceManagerModule.java

```diff
@@ -0,0 +1,86 @@
+package com.robinpowered.RNMDMManager;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+
+// For MDM
+import android.content.RestrictionsManager;
+import android.os.Bundle;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.BroadcastReceiver;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+import com.facebook.react.bridge.WritableNativeMap;
+import java.util.Map;
+import java.util.HashMap;
+import javax.annotation.Nullable;
+
+public class RNMobileDeviceManagerModule extends ReactContextBaseJavaModule {
+    public static final String MODULE_NAME = "MobileDeviceManager";
+    public static final String APP_CONFIG_CHANGED = "react-native-mdm/managedAppConfigDidChange";
+
+    private RestrictionsManager restrictionsManager;
```

Comment:
Ah, meant to place https://github.com/robinpowered/react-native-mdm/pull/1#discussion_r143898160 on this line.

Let's request the system service each time we need it. While this is probably fine the way it is, for a kiosk application running 24/7, it worries me that this reference might change over a prolonged period of time. I'm not sure, but wouldn't mind erring on the side of caution..